generators/chipyard/src/main/resources/csrc/cospike.cc

```diff
@@ -284,9 +287,9 @@ extern "C" void cospike_cosim(long long int cycle,
     printf("%d exception %lx\n", cycle, cause);
   if (valid) {
     printf("%d Cosim: %lx", cycle, iaddr);
-    if (has_wdata) {
-      printf(" s: %lx", wdata);
-    }
+    // if (has_wdata) {
+    //   printf(" s: %lx", wdata);
+    // }
```

Comment:
Nit: Just delete?

Reply:
Rocket-chip is missing an API which will allow us to add these printfs back, 
I'm leaving this in as a reminder for the future